doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -212,6 +212,9 @@ nRF Desktop
     The PDK can act as a sample mouse or keyboard.
     It supports the Bluetooth LE HID data transport and uses SoftDevice Link Layer with Low Latency Packet Mode (LLPM) enabled.
     The PDK uses MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``) and supports firmware updates using the :ref:`nrf_desktop_dfu`.
+  * Support for the nRF54H20 DK with the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
```

Comment:
It would be best either to update the documentation later in one go or add it gradually with each feature (this PR introduces BLE and LLPM).

If you choose the second option (adding it gradually), there are still some missing stuff here (e.g. the nRF Desktop Readme file update).

Reply:
I thought about adding all documentation at once later when more features are merged.

Reply:
so please remove the changelog item as this approach is inconsistent

Reply:
So we shouldn't add entry in changelogs if we do not add doc support?

Reply:
yes, as it is part of doc support. Either you cover everything or add it later as part of a follow-up ticket.